### PR TITLE
Update example apps

### PR DESCRIPTION
**Pull request by sfc-gh-pjafari**
- add missing readme files
- remove "create security integration" step from spcs example as it's no longer needed. See [jira ticker](https://snowflakecomputing.atlassian.net/browse/SNOW-1527720)

### CODE REVIEW

File: spcs-three-tier/setup.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 set -e
-snow sql -f "prepare/spcs_setup.sql"
```
Comment (sfc-gh-melnacouzi) [inline spcs-three-tier/setup.sh:2]: why are we removing this? Maybe some context would help.

Comment (sfc-gh-pjafari) [reply spcs-three-tier/setup.sh:2]: updated PR description with Jira ticket :) 
https://snowflakecomputing.atlassian.net/browse/SNOW-1527720

**Review by sfc-gh-melnacouzi — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -110,6 +110,12 @@ jobs:
             echo “Nothing to test”
           else
             pytest $args
+            ret=$?
+            echo $ret
+            if [ "$ret" = 5 ]; then
+              echo "No tests collected."
+              exit 0
+            fi
```
Comment (sfc-gh-bdufour) [inline .github/workflows/ci.yml:118]: curious: how did you run into this?

Comment (sfc-gh-pjafari) [reply .github/workflows/ci.yml:118]: good question :D not all sample projects have python code/python tests. by default, pytest exits with error code 5 if that's the case. 
The action is still not passing though still so I gotta look into it more (it worked when I tested locally).

File: spcs-three-tier/app/manifest.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ configuration:
 
 artifacts:
   setup_script: setup.sql
+  readme: README.md
```
Comment (sfc-gh-cgorrie) [inline spcs-three-tier/app/manifest.yml:11]: The file actually has a different case (in app/):
```suggestion
  readme: readme.md
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (sfc-gh-bdufour) [reply .github/workflows/ci.yml:118]: I see, the workflow is failing. I'm a bit rusty, does GH pick up the new workflow from the PR, or would you have to merge this first? If it's running the version from `main`, your fix won't take effect yet...

File: spcs-three-tier/app/manifest.yml (hunk 3 above)
Comment (sfc-gh-pjafari) [reply spcs-three-tier/app/manifest.yml:11]: done :D

File: .github/workflows/ci.yml (hunk 2 above)
Comment (sfc-gh-cgorrie) [reply .github/workflows/ci.yml:118]: Also, the CI is known to be broken right now. So no worries about failing runs at the moment, just run the tests locally and confirm they're passing before merge.

**Review by sfc-gh-cgorrie — APPROVED** [review]

**Pull request merged**